exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -123,23 +131,43 @@ class OTLPExporterMixin(
         endpoint: OpenTelemetry Collector receiver endpoint
         credentials: ChannelCredentials object for server authentication
         metadata: Metadata to send when exporting
+        compression: Compression algorithm to be used in channel
     """
 
     def __init__(
         self,
         endpoint: str = "localhost:55680",
         credentials: ChannelCredentials = None,
         metadata: Optional[Tuple[Any]] = None,
+        compression: GRPCCompression = GRPCCompression.NO_COMPRESSION,
     ):
         super().__init__()
 
         self._metadata = metadata
         self._collector_span_kwargs = None
 
+        compression_str = os.environ.get(
```

Comment:
Shouldn't `compression` take priority?

Reply:
I'm not sure, did you see somewhere in the specs where it says that `compression` should take priority?

Reply:
I mean usually the explicitly passed in variable should take priority over the environment variable.

Reply:
@wilguo can you update the precedence here, then we can review the PR and get it merged, thanks!